scc/src/main.rs

```diff
@@ -14,19 +14,30 @@ use redscript_compiler::source_map::{Files, SourceFilter};
 use redscript_compiler::unit::CompilationUnit;
 use serde::Deserialize;
 use time::format_description::well_known::Rfc3339 as Rfc3339Format;
-use time::OffsetDateTime;
+use time::{OffsetDateTime};
 #[cfg(feature = "mmap")]
 use vmap::Map;
 
 fn main() -> Result<(), Error> {
     // the way cyberpunk passes CLI args is broken, this is a workaround
-    let args: Vec<String> = std::env::args().skip(1).collect();
+    let args: Vec<String> = std::env::args().skip(1).collect(); 
     match &args[..] {
         [cmd, path_str, ..] if cmd == "-compile" => {
             let script_dir = PathBuf::from(path_str.split('"').next().unwrap());
             let cache_dir = script_dir.parent().unwrap().join("cache");
-            setup_logger(&cache_dir)?;
-            let manifest = ScriptManifest::load_with_fallback(&script_dir);
+
+            // load manifest without fallback
+            let manifest = ScriptManifest::load(&script_dir);
+
+            // set up logger with an optional manifest
+            setup_logger(&cache_dir, manifest.as_ref().ok().unwrap().append_date_to_logfiles.unwrap())?;
```

Comment:
this will not work as expected, you can't unwrap the manifest here because it could be missing, but you could do this:
```suggestion
            setup_logger(&cache_dir, manifest.as_ref().ok().and_then(|m| m.append_date_to_logfiles).unwrap_or(false))?;
```